### ISSUE TITLE
Arm support inconsistent

### ISSUE THREAD

**Issue by richard-scott** [pyrogram/tgcrypto#1]
Hi,

I'm using a Raspberry Pi 3 Model B with [pyenv](https://github.com/pyenv/pyenv) to manage my Python versions. I can install your package with Python 3.5.4 and get this file installed:

```
~/.pyenv/versions/3.5.4/lib/python3.5/site-packages/tgcrypto.cpython-35m-arm-linux-gnueabihf.so
```
However, when I change to using Python 3.6.6 I get this:
```
~/.pyenv/versions/3.6.6/lib/python3.6/site-packages/tgcrypto.cpython-36m.so
```
Obviously as the filename is different I can't load the module in Python.

**Comment by richard-scott** [comment]
Installation on Python 3.7.0 works fine too.

I donwloaded this [TgCrypto-1.0.4-cp36-cp36m-linux_armv7l.whl](https://files.pythonhosted.org/packages/73/5b/740cccc96dd79b8a3a16cd9eeb095a5246031ff8370e484732641079cde3/TgCrypto-1.0.4-cp36-cp36m-linux_armv7l.whl) and manually checked the contents, and it has the incorrectly named file in there.

**Comment by delivrance** [comment]
@richard-scott Hi there!

You are right, it looks indeed like a faulty wheel.

Unfortunately, I'm not able to do any test on this until next week as I'm on vacation. Meanwhile, you can always compile from source with:

```
pip install tgcrypto --no-binary tgcrypto
```

On your Pi you should already have `gcc` and `python3-dev` packages installed, which are required for the compilation. If not, just install them with `apt` (assuming you are running raspbian).

Soon there also will be an update; I'll keep this open as a reminder.
Thanks for pointing this out!

**Comment by delivrance** [comment]
> Installation on Python 3.7.0 works fine too.

Yeah, the reason being there's no pre-built wheel for 3.7 (yet) and pip automatically builds tgcrypto from source. You can always override this behaviour using the command in my previous comment.

**Comment by delivrance** [comment]
Yes, thanks.
Mostly because I ended up deciding not to upload any wheel for such archs.

The source is always available and error messages hinting about missing compiler/headers are quite self-explanatory.
